docs-next/components/Navigation.tsx

```diff
@@ -61,6 +61,12 @@ export const Navigation = () => {
           Testing
         </NavItem>
       </Section>
+      <Section label="Tutorials">
+        <NavItem href="/tutorials/getting-started-with-create-keystone-next-app">
+          Getting started with `create-keystone-next-app`
```

Comment:
Can we just call this "Getting started"? The long title looks really messy in the side panel.
```suggestion
          Getting started
```

Reply:
The reason for this long title is SEO. I would make a strong case for the title to be as specific as possible ;)

Reply:
My (not necessarily trustworthy) experience here would be that "Getting Started" is OK in the nav, the full phrase should be used in the `<h1>` and `<title>` tags for the page, and Google will get the point.

Another technique websites often use to ensure search engines pay attention to the right phrases is longer descriptions of links in an expanded footer, often de-emphasised from a design point of view. I'm not against doing that per se, but would like to make sure it doesn't look like "icky set optimisation" because I also think that's a turn-off for actual users on the site...

Reply:
Another thing that could be done to help w/ this is ensure this full link exists elsewhere; prominent suggestion would be in the Readme for the package, which will then create an inbound link from both GitHub and NPM

Again, my SEO advice is worth 🧂 but inbound links are generally more important than internal website links, in terms of how search engines will prioritise deciding "this is a relevant page for this term"